lib/chain.js

```diff
@@ -66,6 +66,37 @@ ArrayChain.prototype.fetch = function(fn) {
   return this;
 };
 
+// ArrayChain.prototype.execute = function(chain) {
```

Comment:
Why this is commented out?

Reply:
This was left by mistake, It will be deleted.